lib/http/form_data/multipart.rb

```diff
@@ -12,6 +12,8 @@ module FormData
     class Multipart
       include Readable
 
+      attr_reader :boundary
```

Comment:
As we expose boundary, it worth either return copy of that:

``` ruby
def boundary
  @boundary.dup
end
```

Or at least freeze it in initializer:

``` ruby
def initialize(data)
  # ...
  @boundary = "#{'-' * 21}#{SecureRandom.hex 21}".freeze
end
```

Also, probably, we should allow passing custom boundary value:

``` ruby
def initialize(data, boundary: self.class.generate_boundary)
  # ...
  @boundary = boundary.to_s.freeze
end

def self.generate_boundary
  ("-" * 21) << SecureRandom.hex(21)
end
```